doc/configtables/costs.csv

```diff
@@ -1,15 +1,22 @@
 ,Unit,Values,Description
-year,--,"YYYY; e.g. '2030'","Year for which to retrieve cost assumptions of ``resources/costs.csv``."
-version,--,"vX.X.X; e.g. 'v0.1.0'","Version of ``technology-data`` repository to use."
-rooftop_share,--,float,"Share of rooftop PV when calculating capital cost of solar (joint rooftop and utility-scale PV)."
-USD2013_to_EUR2013, --, "float", "US Dollar-Euro exchange rate. Source: `ECB <ECB: https://www.ecb.europa.eu/stats/exchange/eurofxref/html/eurofxref-graph-usd.en.html>`_."
-fill_values,--,float,"Default values if not specified for a technology in ``resources/costs.csv``."
+year,,"YYYY; e.g. '2030'","Year for which to retrieve cost assumptions of ``resources/costs.csv``."
+technology_data_version,,"vX.X.X; e.g. 'v0.1.0'","Version of ``technology-data`` repository to use."
+country_specific_data,bool,"""{"""",""US""}""",Reference to the desired `technology-data` directory for techno-economic input data
+cost_scenario,,"{"moderate","Only used if `country_specific_data`: ""US""` to select the desired NREL financial case",,
+financial_case,,"{"market","Only used if `country_specific_data: ""US""` to select the desired NREL financial case",
+discountrate,,"{0.071,0.086,0.111}","Rate of return used to discount future cash flows back to their present value"
```

Comment:
for discount rate we support also other values not limited to {0.071,0.086,0.111}, we can say

```suggestion
discountrate,,float,"Rate of return used to discount future cash flows back to their present value"
```